tasks/fractal.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = (gulp, $, pkg) => {
   const path = require('path');
-  const fractal = require(path.resolve(pkg.gulpPaths.fractalConfigPath));
+  const fractal = require(path.resolve(pkg.gulpPaths.fractalConfigFilePath));
```

Comment:
Oh, I didn't see it in action earlier; `pkg.gulpPaths.fractalConfig` would be sufficient then, as `gulpPaths` sufficiently clarifies that all of the subproperties beneath it are filepaths.